app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -46,7 +48,8 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
       return Body.empty;
     }
 
-    final RepositorySlug slug = Config.flutterSlug;
+    final String repo = request!.uri.queryParameters[kRepoParam] ?? 'flutter/flutter';
```

Comment:
Generally, Cocoon refers to `RepositorySlug.full` in string form as `repository`. `repo` refers to the RepositorySlug.name, defaulting the owner to 'flutter'

Reply:
Done.